doc/conf.py

```diff
@@ -56,26 +50,17 @@
 
 # General information about the project.
 project = 'dbxincluder'
-copyright = '2015, Thomas Schraitle'
-author = 'Thomas Schraitle'
+copyright = '2016, Fabian Vogt'
+author = 'Fabian Vogt'
```

Comment:
> It's common sense in the open source world and considered as polite IMHO.

Ok, but only if that means that you won't complain about bugs assigned to you :P
What's the usual convention for referring to multiple authors?